src/script/arch-update.sh

```diff
@@ -195,22 +200,22 @@ icon_dir="/usr/share/icons/${name}"
 
 # Definition of the icon_checking function: Change icon to "checking"
 icon_checking() {
```

Comment:
From my understanding, we are not referring to an icon anymore but to a state (even though it only contains an icon name for now). Maybe we should adapt the function names (and the comment above) accordingly like `state_xxxx`?

Reply:
Yes, good point. I was avoiding making larger changes until we've finished getting the mechanism agreed on, but a rename/recomment is certainly in order.